eng/New-TestResources.cmd

```diff
@@ -0,0 +1,17 @@
+@echo off
+
+REM Copyright (c) Microsoft Corporation. All rights reserved.
+REM Licensed under the MIT License.
+
+setlocal
+
+for /f "usebackq delims=" %%i in (`where pwsh 2^>nul`) do (
+    set _cmd=%%i
+)
+
+if "%_cmd%"=="" (
+    echo Error: PowerShell not found. Please visit https://github.com/powershell/powershell for install instructions.
```

Comment:
Is this to verify whether our build agents have Powershell installed? This is less likely to succeed than just prefacing that Powershell needs to be installed for this to work, `cmd` files require a Windows OS or Windows VM to run.